packages/browser/src/middlewares/PopupWindow.ts

```diff
@@ -144,13 +147,29 @@ class PopupWindow implements Messenger {
         return;
       }
 
-      bridge.once(BridgeEvent.ready, resolve);
-      show({
-        onReminderModalCancel: () => {
-          bridge.removeListener(BridgeEvent.ready, resolve);
-          reject();
-        },
-      });
+      this.successHandler = () => {
+        if (!this?.cancelHandler) return;
+        bridge.removeListener(BridgeEvent.hide, this?.cancelHandler as any);
```

Comment:
```suggestion
        bridge.removeListener(BridgeEvent.hide, this?.cancelHandler as any);
```
不加 as any 還是會 build 不過